modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/MdfList/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
As we will develop the pagination using [SWR](https://swr.vercel.app/docs/pagination). It should be good remove this mocked component and we can develop this together :)